### PR TITLE
Update to preface

### PR DESCRIPTION
**Pull request by tonycurtis**
Spell out what PGAS is, split up the application domain list a bit.  A such-as/like list doesn't need "etc" at the end.  The "this manual"/"the manual" sounded a little repetitive to me.

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1081/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/512/ for details (Mellanox internal link).

**Comment by shamisp** [comment]
@tonycurtis - please add UH copyright.

**Comment by shamisp** [comment]
autobot build

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1082/ for details.

File: doc/doxygen/preface.dox [hunk 1]
```diff
@@ -9,13 +9,13 @@
  * \section Scope Scope of the Document 
  * This document describes the UCX programming
  * interface.  The programming interface exposes a high performance
- * communication API, which provides basic building blocks for PGAS, Message
- * Passing Interface (MPI), Big-Data, Analytics, File I/O, and storage library developers.
+ * communication API, which provides basic building blocks for programming models such as the Partitioned Global Address Space (PGAS) and the Message
```
Comment (yosefe) [inline doc/doxygen/preface.dox]: please split long lines

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1084/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/514/ for details (Mellanox internal link).

**Comment by shamisp** [comment]
@tonycurtis - once you done with this changes please squash the commits.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1095/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/523/ for details (Mellanox internal link).

**Comment by tonycurtis** [comment]
English pass, squashed commits

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1118/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/540/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1123/ for details.

**Comment by abouteiller** [comment]
:+1:

**Comment by shamisp** [comment]
@tonycurtis - can you please fix the merge issue ? Once it is fixed, we will merge it into the master. Thanks !

**Comment by yosefe** [comment]
closing this due to inactivity

**Pull request closed without merging**